cvat/apps/lambda_manager/views.py

```diff
@@ -299,7 +301,11 @@ def invoke(self, db_task: Task, data: Dict[str, Any], *, db_job: Optional[Job] =
                 .format(self.id, str(err)),
                 code=status.HTTP_400_BAD_REQUEST)
 
+        if request:
```

Comment:
Why is this conditioned on a request being supplied? It would be logical to expect a "lambda function call" signal to be sent on all lambda function calls.

Reply:
The `invoke` function calls not only for POST /lambda/function/{id} request but also for POST /lambda/request, but we want to signal only about the first one.